### ISSUE TITLE
[Feature request] Public group chat

### ISSUE THREAD

**Issue by Myppomeu** [qTox/qTox#3023]
Would be great to create public group chats, that available to everyone.
In toxcore issue tracker [people say](https://github.com/irungentoo/toxcore/issues/1490#issuecomment-196098154) that it "included".

**Comment by ProMcTagonist** [comment]
New groupchats will be joinable with an ID, instead of needing a request from a friend. Is that sufficiently "available to everyone" or were you imagining something else? 

You can follow their progress at https://github.com/JFreegman/toxcore/tree/new_groupchats

**Comment by Myppomeu** [comment]
Yes, I think you understand me correct. Saying "available to everyone" I mean that everyone can join chat without request from a friend. For example, public link.
Is that "new groupchats" are what I'm talking about?

**Comment by ProMcTagonist** [comment]
Yep, with new groups you won't need to be friends with any group members to get in.

**Comment by Myppomeu** [comment]
Thanks!
Should I close issue or wait while new groupchats will be ready?

**Comment by ProMcTagonist** [comment]
I can't find any other open issues about your particular question (publicity of the chats) although there are others that are about new groups. I guess it's up to @zetok.
